### ISSUE TITLE
Duplicated Day - Safari

### ISSUE THREAD

**Issue by joseglego** [rotaready/moment-range#192]
Hello,

I'm using this lib. And when I try to get the days from a range. It returns a specific day (August 12, 2017) duplicated. It only happens in Safari. It works in Google Chrome and Firefox. Here you can find a plunkr code which Work in Firefox, Chrome but doesn't in Safari.

https://plnkr.co/edit/wgU0shVb4M7MwyffHNiR?p=preview

**Comment by jontelm** [comment]
Works for me in safari 11 (desktop)
```
console.log('start')
const start = moment('2017-07-10');
const end   = moment('2017-07-15');
const range = moment.range(start, end);
const days = Array.from(range.by('days'));
days.map(function (a) {
  console.log(a.format());
});
console.log('end')
```

[Log] start (script.js, line 3)
[Log] 2017-07-10T00:00:00+02:00 (script.js, line 9)
[Log] 2017-07-11T00:00:00+02:00 (script.js, line 9)
[Log] 2017-07-12T00:00:00+02:00 (script.js, line 9)
[Log] 2017-07-13T00:00:00+02:00 (script.js, line 9)
[Log] 2017-07-14T00:00:00+02:00 (script.js, line 9)
[Log] 2017-07-15T00:00:00+02:00 (script.js, line 9)
[Log] end (script.js, line 11)

**Comment by joseglego** [comment]
Yes @jontelm You are right. But I'm too. It sounds suspicious. 

The short long story: In my country on Aug 12, we had a change of time zone. So, when moment add 1 one day to Aug 12 it's still Aug 12 JUST in Safari, in chrome and mozilla it works. 

I think I should open this issue in `moment` lib. And not here. But I noticed later, because I worked with moment in a handcraft way and did the range with `moment` & `for` and NOT `moment-range`, by my self an it doesn't work.

P.S.: I didn't know about the change of timezone because I moved  recently.
